### DISCUSSION TITLE
IAP

### DISCUSSION THREAD

**Discussion by newuser9290** [LIJI32/SameBoy#627] [Q&A]
Sorry for dumb question I'm on iOS and want to support developer to unlock skins but iOS sends me to iTunes for help. Please help thank you

**Comment by LIJI32** [comment]
Hi, can you share screenshots of the flow you're seeing?

**Reply by newuser9290** [reply]
It finally fixed itself and I am currently subscribed monthly. Thank you for all your hard work. But this is what the error looked like (an example) in case it happens to anyone else in the future ![image](https://github.com/LIJI32/SameBoy/assets/164794137/45a492c1-8d6e-4149-a1af-25875fa30087)

**Comment by LIJI32** [comment]
Odd. Seems to be an issue on Apple's side, glad you got it fixed. Thank you for subscribing!

**Reply by newuser9290** [reply]
Will there ever be able cloud support via iCloud or WebDAV Dropbox etc? I can do a manual backup was just wondering
